python/setup.py

```diff
@@ -1,7 +1,5 @@
-# -*- coding: utf-8 -*-
-
 from setuptools import setup, find_packages
-__version__ = '0.5.1'
+__version__ = '0.5.2'
```

Comment:
is this our python module's version? ought it just match the main revision?

Reply:
I figured it would be released with next version

Reply:
go ahead and set it to the same thing now, and please add the line to `tools/release` to update it when we do a release, otherwise it'll surely go out of sync

Reply:
Not seeing where I'd update `tools/release`. Doesn't seem to check the specific versions of the bindings. If we need to add that, I'd honestly prefer you did that, especially since you created the file and understand how it works better than I do. From what I can determine, it seems it shouldn't matter. Of course no idea where OLD_VERSION $1 is supposed to mean other than possibly the version of libsixel that existed prior to this fork.

Reply:
I did notice the published Pypi version documented is not the same version however. Might need to update the doc.

Reply:
Confirmed: https://pypi.org/project/libsixel-python/ points to 0.5.0. We'll need to submit a new PyPi to get the python module updated. @ctrlcctrlv would have to be one to do it though.